eclipse-scout-core/src/util/dragAndDrop.ts

```diff
@@ -20,7 +20,7 @@ const SCOUT_TYPES = {
 export const dragAndDrop = {
 
   SCOUT_TYPES,
-  DEFAULT_DROP_MAXIMUM_SIZE: 50 * 1024 * 1024, // 50 MiB
+  DEFAULT_DROP_MAXIMUM_SIZE: 50 * 1024 * 1024, // 50 MB
```

Comment:
MiB is correct, MB is wrong. Otherwise we have to change the value of the constant to match megabytes instead of mebibytes.